[2020-06-18 22:24:54,372] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: mis.save_raw 2020-06-19T02:24:35+00:00 [queued]>
[2020-06-18 22:24:54,456] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: mis.save_raw 2020-06-19T02:24:35+00:00 [queued]>
[2020-06-18 22:24:54,457] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 22:24:54,457] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-18 22:24:54,457] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 22:24:54,505] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): save_raw> on 2020-06-19T02:24:35+00:00
[2020-06-18 22:24:54,511] {standard_task_runner.py:53} INFO - Started process 4109 to run task
[2020-06-18 22:24:54,818] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: mis.save_raw 2020-06-19T02:24:35+00:00 [running]> strx
[2020-06-18 22:24:54,896] {logging_mixin.py:112} INFO - Saving raw...
[2020-06-18 22:24:54,924] {python_operator.py:114} INFO - Done. Returned value was: save_raw_file
[2020-06-18 22:24:55,014] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=mis, task_id=save_raw, execution_date=20200619T022435, start_date=20200619T022454, end_date=20200619T022455
[2020-06-18 22:25:04,289] {logging_mixin.py:112} INFO - [2020-06-18 22:25:04,289] {local_task_job.py:103} INFO - Task exited with return code 0
